daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/Value.scala

```diff
@@ -339,11 +339,13 @@ object Value extends CidContainer1[Value] {
 
   sealed abstract class ContractId extends Product with Serializable {
     def coid: String
+    def toBytes: Bytes
```

Comment:
You don't seem to make use of the method in this PR. Is this an unrelated change?

Reply:
It wa ;). V0 and V1 override this.

Reply:
Sorry for the confusion. I missed the `_.bytes` in `hashCommittedContract`. 🤦